src/utils/utils.go

```diff
@@ -899,6 +904,25 @@ func IsConstellationIP(ip string) bool {
 	return false 
 }
 
+func IsTrustedProxy(ip string) bool {
```

Comment:
IPInRange shouldn't be in utils.go for this?
Should I move it?

Reply:
Can I edit IPInRange to return true if ipStr == cidrStr and is a valid IP address instead of error "invalid CIDR address" ?
IPInRange doesn't accept IP as CIDR, but it's equivalent to IP/32 for IPv4 and IP/128 for IPv6

<details><summary>Test IP without mask</summary>
<p>

![image](https://github.com/user-attachments/assets/20e8b738-7187-4600-9d19-1b1a1e8a9162)


</p>
</details> 


<details><summary>Proposition :</summary>
<p>

```go
func IPInRange(ipStr, cidrStr string) (bool, error) {
	ip := net.ParseIP(ipStr)
	if ip == nil {
		return false, fmt.Errorf("parse IP: invalid IP address")
	}

	_, cidrNet, err := net.ParseCIDR(cidrStr)
	if err != nil {
		if ipStr == cidrStr {
			return true, nil
		}
		return false, fmt.Errorf("parse CIDR range: %w", err)
	}

	return cidrNet.Contains(ip), nil
}
```

</p>
</details>